package/Dockerfile

```diff
@@ -75,7 +75,7 @@ RUN mkdir -p /var/lib/rancher-data/local-catalogs/system-library && \
     # in git.rancher.io
     git config --global url."https://github.com/rancher/".insteadOf https://git.rancher.io/ && \
     # Charts need to be copied into the sha256 value of git url computed in https://github.com/rancher/rancher/blob/5ebda9ac23c06e9647b586ec38aa51cc9ff9b031/pkg/catalogv2/git/download.go#L102 to create a unique folder per url
-    git clone -b $CATTLE_CHART_DEFAULT_BRANCH --depth 1 https://git.rancher.io/charts /var/lib/rancher-data/local-catalogs/v2/rancher-charts/4b40cac650031b74776e87c1a726b0484d0877c3ec137da0872547ff9b73a721/ && \
+    git clone -b rancher-v1.28 --depth 1 https://github.com/chiukapoor/charts /var/lib/rancher-data/local-catalogs/v2/rancher-charts/cf3633ed98cd12a6fef63cafe25250982640225938313fb83b553d458f095d28/ && \
```

Comment:
Do we need to merge changes from your charts first? Or this can be replaced with the upstream branch now?

Reply:
These changes are part of the testing commit to test RKE2 and k3s with CAPI v1.5.5. Once https://github.com/rancher/charts/pull/3367 is merged will remove the testing commit 468e0caae53ef686341d9f120be461f09e5b1099